neurovault/apps/statmaps/views.py

```diff
@@ -370,8 +372,21 @@ def view_nidm_results(request, collection_cid, nidm_name):
         else:
             form = NIDMViewForm(instance=nidmr)
 
-    context = {"form": form}
-    return render(request, "statmaps/edit_nidm_results.html.haml", context)
+    # Find the corresponding images
+    nidm_collection = NIDMResultStatisticMap.objects.filter(nidm_results=nidmr)
```

Comment:
It's an artifact from an earlier version that I had, and I left it there if we wanted to do something with the associated collection. It is not used and I can remove it!